docs-src/go/choose-dev-cluster.md

```diff
@@ -0,0 +1,133 @@
+
+- [Local development server](#local-dev-server)
+- [Temporal Cloud](#temporal-cloud)
+- [Self-hosted](#self-hosted-temporal-cluster)
+
+:::info Temporal does not directly run your code
+
+Keep in mind that in every scenario, the “Temporal Platform” does not host and run your Workers (application code).
+It is up to you, the developer, to host your application code.
+The Temporal Platform ensures that properly written code durably executes in the face of platform-level failures.
+
+:::
+
+### Local dev server
+
+**When to use a local development server?**
+
+We recommend using the local development server if you are new to Temporal, or want to start something from scratch and don’t have a self-hosted environment ready or want to pay for a Temporal Cloud account.
+
+The Temporal CLI comes bundled with a development server and provides a fast way to start running Temporal Applications.
+
+However, the local development server does not emit any metrics.
+If you are eager to to set up Cluster-level metrics for performance tuning, we recommend using a self-hosted Cluster or Temporal Cloud.
+
+#### Start the dev server
+
+**How to start a local development server**
+
+If you have successfully installed the Temporal CLI, open a new terminal and run the following command:
+
+```bash
+temporal server start-dev
+```
+
+This command automatically starts the Temporal Web UI, creates a default Namespace, and creates an in-memory database.
+
+The Temporal Web UI serves to [http://localhost:8233](http://localhost:8233/).
+
+For more command details and options, see the [CLI reference](/cli/server/start-dev)
+
+#### Create a custom Namespace
+
+**How to create a new Namespace on the development server**
+
+The development server does automatically create a default Namespace (named "default") when it starts up.
+However, we will create a custom one for our application.
+Since this is something we recommend at a production level, we recommend practicing it with the development server.
+
+Use the `temporal operator namespace create` command using the Temporal CLI to create a new Namespace on the development server.
+
+```bash
+temporal operator namespace create backgroundcheck_namespace
+```
+
+For command details and options, see the [CLI reference](/cli/operator/namespace/create).
+
+### Temporal Cloud
+
+**When to use Temporal Cloud**
+
+If you do not have a Temporal Cloud Account, you can request one using the link on the [get started](https://docs.temporal.io/cloud/get-started) page.
+
+We recommend starting off with Temporal Cloud if you already have a production use case, or need to move a scalable proof of concept into production.
+
+In other words, Temporal Cloud is perfect if you are ready to run at scale and don’t want the overhead of managing your own self-hosted Cluster.
+
+To create a Namespace in Temporal Cloud, follow the instructions in [How to create a Namespace](/cloud/namespaces-create).
```

Comment:
`get started` is lower case but `How to create a Namespace` is upper case.
They're both titles, should they both be capitalized?